setup.py

```diff
@@ -1,38 +1,33 @@
 import codecs
 import os
-import re
 import sys
 
 from setuptools import find_packages, setup
 
-here = os.path.abspath(os.path.dirname(__file__))
 
-
-def read(*parts):
+def read(rel_path):
+    here = os.path.abspath(os.path.dirname(__file__))
     # intentionally *not* adding an encoding option to open, See:
     #   https://github.com/pypa/virtualenv/issues/201#issuecomment-3145690
-    with codecs.open(os.path.join(here, *parts), 'r') as fp:
+    with codecs.open(os.path.join(here, rel_path), 'r') as fp:
         return fp.read()
 
 
-def find_version(*file_paths):
-    version_file = read(*file_paths)
-    version_match = re.search(
-        r"^__version__ = ['\"]([^'\"]*)['\"]",
-        version_file,
-        re.M,
-    )
-    if version_match:
-        return version_match.group(1)
-
-    raise RuntimeError("Unable to find version string.")
+def get_version(rel_path):
+    for line in read(rel_path).splitlines():
+        if line.startswith('__version__'):
+            # __version__ = "0.9"
+            delim = '\"' if '\"' in line else '\''
```

Comment:
Sorry, just noticed this - any reason we're escaping these instead of just `'"'` and `"'"`?

Reply:
(updated)